### PR TITLE
Add places index

### PR DESCRIPTION
**Pull request by StefanMavrodiev**
Add a common page listing places.

### CODE REVIEW

File: src/Controller/Place.php [hunk 1]
```diff
@@ -8,6 +8,19 @@
 
 class Place extends AbstractController
 {
+    public function index(Request $request)
+    {
+        $placeRepo = $this->getDoctrine()->getRepository(\App\Entity\Place::class);
+        $places = $placeRepo->findAll();
```
Comment (angelk) [inline src/Controller/Place.php]: What if we have 10000+ places?
Data should be loaded only if needed, like the index page

File: config/routes.yaml [hunk 2]
```diff
@@ -87,6 +87,10 @@ app_user_download_personal_data:
   path: /{_locale}/user/personal-data-download
   controller: App\Controller\User::downloadPersonalData
 
+app_place:
+  path: /{_locale}/place
```
Comment (angelk) [inline config/routes.yaml]: This path is not allowing access to `/{_locale}/place/new`. Can we change it to something like `/{_locale}/place/list`?

File: translations/messages.bg.php [hunk 3]
```diff
@@ -16,9 +16,13 @@
     'public' => 'публичен',
     'unlisted' => 'скрит',
     'Add pictures' => 'Добави снимки',
+    'Send by' => 'Изпратена от',
 
+    /* TODO: Duplicate translations. */
```
Comment (angelk) [inline translations/messages.bg.php]: todo should be `@TODO`.

File: src/Controller/Place.php (hunk 1 above)
Comment (StefanMavrodiev) [reply src/Controller/Place.php]: Fixed

File: translations/messages.bg.php (hunk 3 above)
Comment (StefanMavrodiev) [reply translations/messages.bg.php]: Fixed

File: config/routes.yaml (hunk 2 above)
Comment (StefanMavrodiev) [reply config/routes.yaml]: Changed to /{_locale}/places.

**Pull request merged**